Sources/SwiftParser/Directives.swift

```diff
@@ -137,6 +141,7 @@ extension Parser {
         } else {
           condition = nil
         }
+        unexpectedBetweenConditionAndElements = self.consumeRemainingTokenOnLine()
       case .pound:
         if self.atElifTypo() {
```

Comment:
Is this some legacy in the codebase?

Reply:
No, this was recently introduced by https://github.com/apple/swift-syntax/pull/1358